ansible/roles/test/tasks/snmp.yml

```diff
@@ -8,24 +8,28 @@
   assert: { that: "ansible_sysname == '{{ inventory_hostname }}'" }
 
 - block:
-    - name: include snmp cpu test
-      include: roles/test/tasks/snmp/cpu.yml
-
-    - name: inlcude snmp physical table test
-      include: roles/test/tasks/snmp/phys_table.yml
-
     - name: include snmp interfaces test
       include: roles/test/tasks/snmp/interfaces.yml
 
     - name: include snmp pfc counter test
       include: roles/test/tasks/snmp/pfc_counters.yml
 
     - name: include snmp queues test
-      include: roles/test/tasks/snmp/queues.yml
-
-    - name: include snmp PSU test
-      include: roles/test/tasks/snmp/psu.yml
+      include: roles/test/tasks/snmp/queues.yml 
 
     - name: include snmp lldp test
       include: roles/test/tasks/snmp/lldp.yml
   when: testcase_name is defined
+
+- block:
+    - name: include snmp cpu test
+      include: roles/test/tasks/snmp/cpu.yml 
```

Comment:
why cpu test is not available in simx?

Reply:
the tests seems not to working mostly does not look that it does what it aims to do
More investigation is required and will be done regardless of this PR.